library_generation/cli/entry_point.py

```diff
@@ -51,6 +51,15 @@ def main(ctx):
     metadata about library generation.
     """,
 )
+@click.option(
+    "--includes",
+    type=str,
+    default=None,
+    show_default=True,
+    help="""
+    A list of library names that will be generated, separated by comma.
```

Comment:
Can we add more docs here? I think it is basically `library_name != null ? library_name : api_shortname` from `generation_config.yaml`?

Reply:
Changed the comment.